### PR TITLE
Bug fix subscriptions list counter

### PR DESCRIPTION
**Pull request by Tilesto**
screenshot_1
screenshot_2
screenshot_3

### CODE REVIEW

**Comment by Tilesto** [comment]
screenshot_2
screenshot_1

File: VirtoCommerce.SubscriptionModule.Web/Scripts/blades/subscription-list.tpl.html [hunk 1]
```diff
@@ -1,18 +1,18 @@
-﻿<div class="blade-static">
+<div class="blade-static subscription-list">
 	<div class="form-group">
         <div class="form-input __search">
             <input placeholder="{{'platform.placeholders.search-keyword' | translate}}" ng-model="filter.keyword" ng-keyup="$event.which === 13 && filter.criteriaChanged()" style="width: 190px">
```
Comment (n2pro) [inline VirtoCommerce.SubscriptionModule.Web/Scripts/blades/subscription-list.tpl.html]: move this inline style to subscription.css

**Comment by tatarincev** [comment]
@n2pro please review this PR

File: VirtoCommerce.SubscriptionModule.Web/VirtoCommerce.SubscriptionModule.Web.csproj [hunk 2]
```diff
@@ -133,6 +133,7 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Content\css\subscription.css" />
```
Comment (n2pro) [inline VirtoCommerce.SubscriptionModule.Web/VirtoCommerce.SubscriptionModule.Web.csproj]: subscription.css is empty

**Review by n2pro — APPROVED** [review]

**Review by tatarincev — APPROVED** [review]

**Pull request merged**